README.md

```diff
@@ -43,6 +43,15 @@ This project presents a cutting-edge freelancing platform that connects freelanc
    ```sh
    npm install
    ```
+3. Add `.env.local` file under the `quolance-ui` directory.
+   It should contain the following:
+   ```plaintext
+   NEXT_PUBLIC_BASE_URL=http://localhost:8080
```

Comment:
should be 8081 as the server port is set to 8081 in main branch

Reply:
It is only 8081 if you don't run the caddy reverse proxy server, if you're running the server, your request will be redirected to 8081.